data/participants.json

```diff
@@ -10,5 +10,15 @@
 			"twitter"  : "",
             "referral" : ""
         }
```

Comment:
You're missing a comma after the closing bracket on line 12. @Archies11

Reply:
You can use [JSON Linter](https://jsonlint.com) to verify that you have a vallid JSON.